test/plugin/scenarios/undertow-scenario/support-version.list

```diff
@@ -0,0 +1,39 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+1.3.0.Final
+1.3.17.Final
+1.3.33.Final
+1.4.0.Final
+1.4.12.Final
+1.4.27.Final
+2.0.0.Final
+2.0.3.Final
+2.0.4.Final
+2.0.8.Final
+2.0.9.Final
+2.0.11.Final
+2.0.13.Final
+2.0.15.Final
+2.0.16.Final
+2.0.17.Final
+2.0.19.Final
+2.0.20.Final
+2.0.21.Final
+2.0.22.Final
+2.0.23.Final
+2.0.26.Final
+2.0.27.Final
```

Comment:
Could you add some comments and descriptions here about why we removed some versions to be tested? And how we select these versions?

Reply:
done